schema/crdb/21.0.0/up5.sql

```diff
@@ -0,0 +1,12 @@
+-- Preemptively turn any former fleet defaults into non-defaults if there's
+-- going to be a silo conflicting with it after up6
+UPDATE omicron.public.ip_pool_resource AS ipr
+SET is_default = false
+FROM omicron.public.ip_pool as ip
+WHERE ipr.resource_type = 'silo' -- technically unnecessary because there is only silo
+  AND ipr.is_default = true
+  AND ip.is_default = true -- both being default is the conflict being resolved
+  AND ip.silo_id IS NOT NULL
```

Comment:
Should this be `AND ip.silo_id IS null` b/c that's what indicates a fleet ip pool?

Reply:
Nope, we're looking at silo pools in `ip_pool`, lining them up against entries in `ip_pool_resource` that are type=silo but came from a fleet pool in up4, preferring the former as a default.